laser_encoders/models.py

```diff
@@ -83,6 +91,11 @@ def __init__(
         self.encoder.eval()
         self.sort_kind = sort_kind
 
+    def __call__(self, sentences):
```

Comment:
Can you please update the documentation with a mention of this method?

Reply:
done